osu.Game/Overlays/Profile/Sections/Ranks/DrawableProfileScore.cs

```diff
@@ -98,7 +98,7 @@ private void load(RulesetStore rulesets)
                                                         Font = OsuFont.GetFont(size: 12, weight: FontWeight.Regular),
                                                         Colour = colours.Yellow
                                                     },
-                                                    new DrawableDate(Score.Date, 12)
+                                                    new DrawableDate(Score.EndedAt ?? DateTimeOffset.Now, 12)
```

Comment:
`DateTimeOffset.Now` isn't really a good fall back if the score can't provide the date. At the very least it should not add a drawable date at all if missing.

Reply:
I did this match behaviour in another place where `SoloScoreInfo` is used. I don’t remember where but I did it to remain consistent. I can change it if you want.

Reply:
It is used in `SoloScoreInfo.ToScoreInfo`, correct, but it doesn't seem like that method had any better way to handle null `ended_at` value.

Reply:
I've made this change in #19210. Seems to be okay.